easybuild/easyconfigs/p/PETSc/PETSc-3.7.5-intel-2016a-download.eb

```diff
@@ -0,0 +1,41 @@
+name = 'PETSc'
+version = '3.7.5'
+
+homepage = 'http://www.mcs.anl.gov/petsc'
+description = """PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
+ scalable (parallel) solution of scientific applications modeled by partial differential equations."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['http://ftp.mcs.anl.gov/pub/petsc/release-snapshots']
+sources = [SOURCELOWER_TAR_GZ]
+
+shared_libs = 1
+pic = 1
+
+downloadinstall = True
+
+dependencies = [('HDF5','1.8.18')] 
+
+configopts = ""
+# compiles PETSc shared
+configopts += "--with-mkl_pardiso=1 --with-mkl_pardiso-dir=$MKLROOT "
+configopts += "--with-hdf5=1 --with-hdf5-dir=$EBROOTHDF5"
+
+# external packages that are downloaded
+external_packages = ['mumps', 'ptscotch', 'superlu', 'superlu_dist', 'parmetis', 'metis', 'ml', 'suitesparse', 'hypre', 'prometheus']
```

Comment:
line too long, should be < 120 characters

also, please sort these alphabetically since order doesn't matter?

```python
external_packages = ['hypre', 'metis', 'ml', 'mumps', 'parmetis', 'prometheus', 'ptscotch',
                     'suitesparse', 'superlu', 'superlu_dist']
```